x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/__snapshots__/trusted_apps_page.test.tsx.snap

```diff
@@ -0,0 +1,5573 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Should this snapshot file be deleted? It looks unrelated to the changes.

Reply:
This snapshot was deleted on master and supposed to be removed during a merge but because of a conflict it didn't get properly removed from my branch. Fixed.